### ISSUE TITLE
2.0 Summary

### ISSUE THREAD

**Issue by ilhan007** [SAP/ui5-webcomponents#8608]
# Release Timelines
- Release `2.0` RC versions in April 
   - [2.0.0-rc.0](https://github.com/SAP/ui5-webcomponents/releases/tag/v2.0.0-rc.0) (2024-04-09)
   - [2.0.0-rc.1](https://github.com/SAP/ui5-webcomponents/releases/tag/v2.0.0-rc.1) (2024-04-11)
   - [2.0.0-rc.2](https://github.com/SAP/ui5-webcomponents/compare/v2.0.0-rc.1...v2.0.0-rc.2) (2024-04-18)
   - [2.0.0-rc.3](https://github.com/SAP/ui5-webcomponents/compare/v2.0.0-rc.2...v2.0.0-rc.3) (2024-05-10)
- Release `2.0` between `mid June - mid July`

# Migration Support
- [Migration guide](https://sap.github.io/ui5-webcomponents/nightly/docs/Migrating%20to%20version%202.0%20guide/)
- Blog post to be provided
- Dev support from UI5 Team if required

# Maintenance of 1.x
- `Support until the end of the year` (approx. 6 months)
- Downport of bug fixes only

# Features
#### New Components

- [x] Dynamic Page
- [x] Form
- [x] Text
- [x] Tokenizer

####  Components Enhancements

- [x] List Drag and Drop (comes with TabContainer)?
- [x] Tree Drag and Drop
- [x] TabContainer Drag and Drop
- [x] New Notifications Experience

#### Reusable component CSS styles for creating patterns/layouts or other common styling

**Use-case:** things that are not feasible to be implemented by creating new components or implemented inside the components

- [x] Input Icon - to make a slotted icon interactive (press, hover states) by applying a single CSS class
- [x] Link Icon - to make a slotted icon aligned and styled with the link's text by applying a single CSS class

#### New Site for Documentation, Samples and API reference
See the [preview](https://sap.github.io/ui5-webcomponents/nightly/).

- Live editor
- Playground
- Dark mode
- Icons view

# Breaking changes
See the full list [here](https://github.com/SAP/ui5-webcomponents/issues/8461).

- 90% is pure API renaming for better consistency 
- several APIs are replaced by alternatives (listed below)
- few APIs are removed without alternatives (listed below)

### Rename of property, event, enumeration values
~35

### Replaced API

- Methods `openPicker`, replaced by `open/opener` declarative API

- Grouping of Items
Instead of using `ui5-li-groupheader` as separator in a flat structure:
```html
<ui5-list>
	<ui5-li-groupheader>group<ui5-li-groupheader/>
	<ui5-li></ui5-li>
	<ui5-li></ui5-li>
	<ui5-li></ui5-li>
</ui5-list>
```
the API will support true nesting of ListItems inside ListItemGroup:
```html
<ui5-list>
	<ui5-li-group header-text="group">
		<ui5-li></ui5-li>
		<ui5-li></ui5-li>
		<ui5-li></ui5-li>
	<ui5-li-group>
</ui5-list>
```
to easier bind hierarchical data model to the UI.

- Menu: property `starts-section` replaced by `ui5-menu-separator` component.
- List: `property image`, replaced by `slot image`
- TabContainer: `show-overflow` property, replaced by `slot overflowButton`.
- IllustratedMessage: `property titleLevel`, replaced by `slot title`  (that you can set the level on).

#### Progress Indicator (ui5-progress-indicator)
`Remove disabled property` - Progress indicator isn't an interactive element so disabled property make no sense. 
Also aria-disabled is deprecated on the progressbar role since ARIA 1.2 .

#### Selects's Option (ui5-option)
Remove `disabled` property - SAP UX and ACC standards suggest to not include any disabled items in the dropdown.

#### Popover (ui5-popover)
Remove the `modal` and `hide-backdrop` properties -  SAP Design confirmed that these props and behaviour don't belong to the Popover.

#### Token (ui5-token)
Remove the `readonly` property - SAP Design Team recommends to not offer the readonly state on Token level, as it only makes sense when multiple Tokens are used inside `Tokenizer` web component - the `readonly` state will be handled on `Tokenizer` level.

#### TabContainer (ui5-tabcontainer)
Remove `fixed` property as the behaviour is not backed with specs - that TabContainer is not meant to be expanded/collapsed like a Panel.

**Comment by henriquemattos** [comment]
Thank you for all the information, transparency, and hard-work of the team. Looking forward to v2 ☺️

**Comment by ilhan007** [comment]
Hello Henrique @henriquemattos , thank you for the kind words! 

Yes, we will definitely add the possibility to browser the 1.x version docs, most likely via dropdown in the navigation bar.

**Comment by ilhan007** [comment]
Closing the issue as 2.0 has been released - see the [blog](https://sap.github.io/ui5-webcomponents/blog/releases/announcing-v2/) post what's new!

Consider the following URLs for UI5 Web Components Documentation
 https://sap.github.io/ui5-webcomponents/ - the latest released (2.0.1 as of now)
 https://sap.github.io/ui5-webcomponents/nighlty/ - the version under development (that will be released)
 https://sap.github.io/ui5-webcomponents/v1/ - the version 1 (1.24.7)
  
 Additionally, you can reach the above version via the dropdown in the documentation website: 
<img width="223" alt="Screenshot 2024-07-08 at 15 19 04" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/bc041bb9-2b8a-47fc-80a8-c88512154da0">

FYI @henriquemattos
